src/github.com/getlantern/flashlight/config/flags.go

```diff
@@ -89,3 +100,14 @@ func (updated *Config) applyFlags() error {
 
 	return nil
 }
+
+func parseFrontFQDNs(frontFQDNs string) (map[string]string, error) {
+	fqdns := map[string]string{}
+	if err := yaml.Unmarshal([]byte(frontFQDNs), fqdns); err != nil {
+		return nil, err
+	}
+	if len(fqdns) == 0 {
```

Comment:
@oxtoacart: we are checking here for an empty set of fqdns.  This error gets propagated up to the call to `config.Start` in `flashlight.go:main`, where it triggers `golog.Fatal`, exiting the program:

```go
	if err != nil {
		log.Fatalf("Unable to start configuration: %s", err)
	}
```
That's why I believe we should never reach the condition below, where we have an empty mapping of fqdns.